Replicator/Replicator.cc

```diff
@@ -695,14 +695,14 @@ namespace litecore { namespace repl {
         if ( _connectionState != Connection::kConnected )
             return;  // Not ready yet; Will be called again from _onConnect.
 
-        for ( int i = 0; i < _subRepls.size(); i++ ) {
-            if ( !_subRepls[i].remoteCheckpointDocID )
-                _subRepls[i].remoteCheckpointDocID = _subRepls[i].checkpointer->initialCheckpointID();
+        for ( auto& _subRepl : _subRepls ) {
```

Comment:
`const auto&` ?  If the checkpoint ID methods are not const, though, then nevermind.

Reply:
It can't be because of the assignment on line 700